src/index.scss

```diff
@@ -1,8 +1,17 @@
-@import "./animations.scss";
-@import "src/Themes/all.scss";
-@import "./border-radius.scss";
-@import "./colors.scss";
-@import "./content-colors.scss";
-@import "./shadows.scss";
-@import "./spacing.scss";
-@import "./typography.scss";
\ No newline at end of file
+// MONDAY VIBE DESIGN SYSTEM : DESIGN TOKENS
```

Comment:
theoretically, `monday-ui-style` can be used without Vibe. are we sure we want to "bind" the two libraries?

Reply:
* Basically its Vibe's tokens. so i am not sure it should be decoupled from the name. 
* per now It also includes the icons who should be separated from this repo to my opinion.
* If you want to go further, it should be a package alongside monday-react package and monday-compoenents packages or something similar. this should be planned separately.